lib/role_control/pundit_interop.rb

```diff
@@ -0,0 +1,11 @@
+module RoleControl
+  module PunditInterop
+    extend ActiveSupport::Concern
+
+    module ClassMethods
+      def scope_for(action, api_user, opts={})
```

Comment:
You may have explained this in the next PR but I'm a little confused as to why this interop exists. Why does a model using the pundit controller need the to define it's own `scope_for` method is this for link management in the `organizations_controller` or just the other linking model controllers? If so i get this then though it was confusing why we access pundit through two different channels, i.e. model -> pundit and controller->pundit.